Doc/whatsnew/3.14.rst

```diff
@@ -124,7 +124,8 @@ Add notes for JSON serialization errors that allow to identify the source
 of the error.
 (Contributed by Serhiy Storchaka in :gh:`122163`.)
 
-Enable :mod:`json` module to work as a script using the :option:`-m` switch.
+Enable :mod:`json` module to work as a script using the :option:`-m` switch: ``python -m json``.
+See :ref:`JSON command-line interface<json-commandline>`.
```

Comment:
This feels like it is missing an article (and improve some spacing). I'm not fully sure of this though, so alternative proposals might be better.
```suggestion
See the :ref:`JSON command-line interface <json-commandline>`.
```

Reply:
While it doesn't read quite right in the plain text form, it still works in HTML because of the hyperlink. Wrapping the link in a noun phrase can make it read right in both formats:

```suggestion
See the :ref:`JSON command-line interface<json-commandline>` documentation.
```

I don't think it's necessary to change, though (since most folks will be reading the HTML version).

Reply:
We definitely want to add the missing space even if the wording is left alone:

```suggestion
See :ref:`JSON command-line interface <json-commandline>`.
```

Reply:
Added the space and the words `the` and `documentation` 👍